views/404.handlebars

```diff
@@ -0,0 +1,10 @@
+<link rel="stylesheet" href="/assets/css/404.css">
+
+{{> header}}
+
+<div class="wrapper">
+    <div class="status"><b>404</b></div>
+    <div class="description">{{ locale.notfound.description }}</div>
+    <img src="/assets/images/shockedbandwidth.svg" class="shocked-bandwidth"/>
+    {{> footer }}
```

Comment:
Tabs
```suggestion
	<div class="status"><b>404</b></div>
	<div class="description">{{ locale.notfound.description }}</div>
	<img src="/assets/images/shockedbandwidth.svg" class="shocked-bandwidth"/>
	{{> footer }}
```

Reply:
Hmm I will fix it but I find it odd that my IDE (at least on HTML)  replaces tab with spaces (since I pressed tab myself)